app/src/main/java/org/oppia/android/app/help/thirdparty/LicenseTextViewerActivityPresenter.kt

```diff
@@ -44,8 +46,12 @@ class LicenseTextViewerActivityPresenter @Inject constructor(
       (activity as LicenseTextViewerActivity).finish()
     }
 
-    binding.licenseTextViewerActivityToolbarTitle.setOnClickListener {
-      binding.licenseTextViewerActivityToolbarTitle.isSelected = true
+    activity.applicationContext?.let {
```

Comment:
Here and everywhere else with the change, remove the application context reference since accessibilityService [handles its own context](https://github.com/oppia/oppia-android/blob/d049a5a8ba2423802391475f355f99880b479c48/utility/src/main/java/org/oppia/android/util/accessibility/AccessibilityServiceImpl.kt#L10)